__tests__/integration/configuration/test-constants.js

```diff
@@ -73,7 +73,7 @@ export const TOKEN_DATA = {
    * @param {number} tokenData "token_data" property from an output
    * @returns {boolean} True if this is an authority output
    */
-  isAuthorityToken: tokenData => transactionUtils.isTokenDataAuthority(tokenData)
+  isAuthorityToken: tokenData => (libConstants.TOKEN_AUTHORITY_MASK & tokenData) > 0,
```

Comment:
We don't have a method like this in the lib anymore? Maybe it's worth having it

Reply:
We already have a method for this. But I think the best solution would be to remove the `isAuthorityToken` altogether since we can use the lib method on the tests instead. What do you think?